parquet/src/file/metadata.rs

```diff
@@ -940,6 +974,38 @@ impl OffsetIndexBuilder {
     }
 }
 
+/// Metadata for sorting order for columns
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct SortingColumnMetaData {
+    /// The column index (in this row group) *
+    pub column_index: i32,
+    /// If true, indicates this column is sorted in descending order. *
+    pub descending: bool,
+    /// If true, nulls will come before non-null values, otherwise,
+    /// nulls go at the end.
+    pub nulls_first: bool,
+}
+
+impl From<&SortingColumn> for SortingColumnMetaData {
+    fn from(sorting_column: &SortingColumn) -> Self {
+        SortingColumnMetaData {
```

Comment:
```suggestion
impl From<SortingColumn> for SortingColumnMetaData {
    fn from(sorting_column: SortingColumn) -> Self {
        SortingColumnMetaData {
```

As these are copy types